### PR TITLE
proposal for link age macros

### PR DESCRIPTION
**Pull request by playerskel**
Another small macro change that makes reading the code just a bit less magic.

### CODE REVIEW

File: include/macros.h [hunk 1]
```diff
@@ -31,8 +31,11 @@
 
 #define GET_ACTIVE_CAM(globalCtx) ((globalCtx)->cameraPtrs[(globalCtx)->activeCamera])
 
-#define LINK_IS_ADULT (gSaveContext.linkAge == 0)
-#define LINK_IS_CHILD (gSaveContext.linkAge == 1)
+#define LINK_ADULT 0
+#define LINK_CHILD 1
```
Comment (Dragorn421) [inline include/macros.h:35]: I would put this is z64save.h (where `linkAge` is defined), and make it a `LinkAge` enum. Also prefix should probably be `LINK_AGE_`

The comment on `SaveContext#linkAge` could then be removed entirely (enums for specific fields often have the same name as the field)

Comment (playerskel) [reply include/macros.h]: Makes sense. Let me know if this is better.

**Review by Dragorn421 — APPROVED** [review]
Looks good to me, unless there's a reason this hasn't been done before

**Review by fig02 — APPROVED** [review]
i feel like theres possibly a different approach for age stuff that we were going to take, but i cant remember it if it ever existed or if im confusing it with something else. 
im fine with this either way

**Review by Roman971 — APPROVED** [review]

**Pull request merged**